### ISSUE TITLE
Brave browser: NVDA doesn't read contextual menu to fill user/password

### ISSUE THREAD

**Issue by iubito** [nvaccess/nvda#20388]
I don't know if it is a Brave or NVDA issue, as it appears after both updates recently. Tested on current version and beta version of NVDA.

When logging in a website, a form ask for username and password. Saved user/pass are shown in a contextual menu when I press the arrow down key.

Before the bug appeared, NVDA said "password for MyUserName1", "password for myUserName2".
Since the bug appeared, NVDA say "vide" (empty in French).

**Comment by iubito** [comment]
I don't understand how, I choosed the "bug report" template, but I got less questions as expected.

Tested on Windows 10 x64, latest Brave and latest NVDA stable versions.

What I expect : the previous behaviour, before the bug appeared.

**Comment by seanbudd** [comment]
Please fill out the bug report template in full:
https://github.com/nvaccess/nvda/issues/new?template=01-bug_report.yaml
